admin-company-service/src/controllers/companyController.ts

```diff
@@ -21,7 +21,7 @@ const companyController = {
 
         // destructing the variables
         const { email, name, city, state, phone, logo, status, company_address } = req.body;
-        let message = constants.checkLanguageSetting(req.headers)
+        let message = req.user ? req.user.message ? req.user.message : null : null
```

Comment:
```suggestion
        let message = const message = req.user && req.user.message || null;
```

this will have the same impact I think do check it once and let me know @PoojaQodic